src/main/java/Duke.java

```diff
@@ -14,43 +14,88 @@ public void start() {
                 + "\t |____/ \\__,_|_|\\_\\___|\n";
         System.out.println(logo);
         System.out.println("\t Hello! I'm Duke");
-        System.out.println("\t What can I do for you?");
+        System.out.println("\t What can I do for you?\n");
+    }
+
+    public void display() {
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n");
     }
 
     public void mark(String command) {
-        int taskindex = Integer.parseInt(command.split(" ")[1]) - 1;
-        tasks[taskindex].mark();
+        int index = Integer.parseInt(command.split(" ")[1]) - 1;
+        tasks[index].mark();
     }
 
     public void unmark(String command) {
-        int taskindex = Integer.parseInt(command.split(" ")[1]) - 1;
-        tasks[taskindex].unmark();
+        int index = Integer.parseInt(command.split(" ")[1]) - 1;
+        tasks[index].unmark();
     }
 
     public void list() {
-        for (int i = 0; i < size; i++) {
-            System.out.print("\t " + (i + 1) + ".");
-            tasks[i].showTask();
+        for (int i = 0; i < taskIndex; i++) {
+            System.out.print("\t " + (i + 1) + "." + tasks[i].toString() + "\n");
         }
+        System.out.print("\n");
+    }
+
+    public void addTask(Task task) {
+        tasks[taskIndex] = task;
+        taskIndex++;
+        System.out.println("\t Got it. I've added this task:\n"
```

Comment:
I like what you did here - indenting and spacing a sentence into parts that are more readable!